plugins/author-name/client/containers/AuthorName.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import AuthorName from '../components/AuthorName';
+import {gql} from 'react-apollo';
+import {withFragments} from 'plugin-api/beta/client/hocs';
+
+class AuthorNameContainer extends React.Component {
+  constructor() {
+    super();
+
+    this.state = {
+      tooltip: false
+    };
+
+  }
+
+  showTooltip = () => {
+    this.setState({
+      tooltip: true
+    })
+  }
+
+  hideTooltip = () => {
+    this.setState({
+      tooltip: false
+    })
+  }
+
+  render() {
+    return <AuthorName 
+      {...this}
```

Comment:
Avoid `{...this}` this just passes everything, even the `render` function. It's better to be explicit, so what is passed down is clearly defined: 

```js
<AuthorName 
comment={this.props.comment}
showTooltip={this.showTooltip}
hideTooltip={this.hideTooltip}
tooltipVisible={this.state.tooltip}
/>
```